recipe/meta.yaml

```diff
@@ -67,7 +67,7 @@ test:
   commands:
     - $R -e "library('gdalraster')"  # [not win]
     - $R -e "testthat::test_package('gdalraster', stop_on_failure=TRUE)"  # [not win]
-    - "\"%R%\" -e \"library('gdalraster')\""  # [win] 
+    - "\"%R%\" -e \"library('gdalraster')\""  # [win]
     - "\"%R%\" -e \"testthat::test_package('gdalraster', stop_on_failure=TRUE)\""  # [win]
```

Comment:
The testthat tests aren't running. Maybe we can clean this up in this PR as well?

```suggestion
    - r-codetools
  source_files:
    - tests/
  commands:
    - $R -e "library('gdalraster')"                                          # [not win]
    - $R -e "testthat::test_file('tests/testthat.R', stop_on_failure=TRUE)"  # [not win]
    - "\"%R%\" -e \"library('gdalraster')\""                                          # [win]
    - "\"%R%\" -e \"testthat::test_file('tests/testthat.R', stop_on_failure=TRUE)\""  # [win]
```

Reply:
Thanks @mfansler. I'll do this manually as when I try to apply your suggested changes GitHub is giving me a strange

```
Applying suggestions on deleted lines is currently not supported
```

which I've never seen before. :?

<img width="1414" height="1061" alt="image" src="https://github.com/user-attachments/assets/555d5ded-d539-4c4e-9fef-95e23fc75167" />

Reply:
```
> testthat::test_package('gdalraster', stop_on_failure=TRUE)
> 
> 
No installed testthat tests found for gdalraster
+ exit 0
```

Seems like a pretty bad design choice on `testthat` to have no files found return an exit code of `0`. ☹️